### PR TITLE
Minu

### PR DESCRIPTION
**Pull request by Minuja23**

### CODE REVIEW

File: defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java [hunk 1]
```diff
@@ -33,7 +33,28 @@
 	private String contactNumber;
 	private String nic;
 	private String image;
+	private String verification;
+	private String token;
 	
+	public String getToken() {
+		return token;
+	}
+
```
Comment (Sniththilan) [inline defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java:43]: remove the empty line

File: defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java [hunk 2]
```diff
@@ -33,7 +33,28 @@
 	private String contactNumber;
 	private String nic;
 	private String image;
+	private String verification;
```
Comment (Sniththilan) [inline defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java:37]: write the validation

File: defect-tracker-server/bin/.gitignore [hunk 3]
```diff
@@ -0,0 +1,40 @@
+HELP.md
```
Comment (Sniththilan) [inline defect-tracker-server/bin/.gitignore:1]: remove the bin folder

File: defect-tracker-server/bin/src/main/resources/ValidationMessages.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
+validation.email.alreadyExist=10000
```
Comment (Sniththilan) [inline defect-tracker-server/bin/src/main/resources/ValidationMessages.properties:1]: check the validation code

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java [hunk 5]
```diff
@@ -10,8 +10,10 @@
 import org.springframework.stereotype.Service;
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java:13]: remove the empty line

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java [hunk 6]
```diff
@@ -131,4 +131,20 @@ public String emailVerification(String token, String email) {
...
+		
+	}
+
+	@Override
+	public void updateEmployeeStatus(Login login) {
+		// TODO Auto-generated method stub
+		
+	}
+	
+
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java]: remove the empty lines

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java [hunk 7]
```diff
@@ -131,4 +131,20 @@ public String emailVerification(String token, String email) {
...
+	@Override
+	public void create(Login login) {
+		loginRepository.save(login);
+		
+	}
+
+	@Override
+	public void updateEmployeeStatus(Login login) {
+		// TODO Auto-generated method stub
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java]: if you do not need this method remove it

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java [hunk 8]
```diff
@@ -5,6 +5,8 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.sql.Date;
+
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:8]: remove the empty  line

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java [hunk 9]
```diff
@@ -6,6 +6,8 @@
 import java.util.Optional;
 import java.util.UUID;
```
Comment (Sniththilan) [inline defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java:8]: remove the empty  line

Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java:8]: removed and customized

File: defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java (hunk 1 above)
Comment (Minuja23) [reply defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java:43]: cleaned and removed

File: defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java (hunk 2 above)
Comment (Minuja23) [reply defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java:37]: no need to validate on entity it is validated on DTO

File: defect-tracker-server/bin/.gitignore (hunk 3 above)
Comment (Minuja23) [reply defect-tracker-server/bin/.gitignore:1]: bin folder removed

File: defect-tracker-server/bin/src/main/resources/ValidationMessages.properties (hunk 4 above)
Comment (Minuja23) [reply defect-tracker-server/bin/src/main/resources/ValidationMessages.properties:1]: validation codes changed and commited

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java (hunk 5 above)
Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java:13]: cleared

File: defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java (hunk 7 above)
Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java]: removed in the next commit

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java (hunk 8 above)
Comment (Minuja23) [reply defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:8]: removed

**Pull request merged**